Jenkinsfile

```diff
@@ -38,16 +38,20 @@ spec:
     }
 
     parameters {
-        string(name: "CW_CLI_BRANCH", defaultValue: "master", description: "Codewind CLI branch from which to download the latest build")
-        string(name: "APPSODY_VERSION", defaultValue: "0.4.3", description: "Appsody executable version to download")
+        string(name: "APPSODY_VERSION", defaultValue: "0.4.4", description: "Appsody executable version to download")
     }
 
     stages {
         stage("Download dependency binaries") {
             steps {
                 dir("dev/bin") {
                     sh """#!/usr/bin/env bash
-                        export CW_CLI_BRANCH=${params.CW_CLI_BRANCH} APPSODY_VERSION=${params.APPSODY_VERSION}
+                        # Use the current branch if not a PR build, use master for PR builds
```

Comment:
I'm guessing this change is the reason the PR build broke?
```
16:24:18  ++ dirname ./pull.sh
16:24:18  + cd .
16:24:18  + ./cli-pull.sh
16:24:18  Downloading latest Codewind CLI built from PR-210
16:24:18  Downloading https://download.eclipse.org/codewind/codewind-installer/PR-210/latest/build_info.properties
16:24:18  curl: (22) The requested URL returned error: 404 
16:24:18  grep: cli_version.properties: No such file or directory
16:24:18  Latest cli PR-210 build is 
16:24:18  grep: cli_version.properties: No such file or directory
16:24:18  Downloading https://download.eclipse.org/codewind/codewind-installer/PR-210/latest/codewind-installer-linux
16:24:18  curl: (22) The requested URL returned error: 404 
```

Reply:
This also assumes there will always be a corresponding installer branch for each codewind-vscode branch, which may not be the case -- for example, Howard's `hybrid` branch.

Reply:
ya, i was thinking of 0.4.0 for example, but this won't work as written